src/components/ProjectMeta.tsx

```diff
@@ -80,20 +82,26 @@ const ProjectMeta: React.FunctionComponent<ProjectMetaProps> = ({project}) => {
       <div className="row">
         <span className="label">Tech Stack</span>
         <div className="content">
-          {
-            project.techstack.map(tech => (
-              <Link to={`/tech/${_.kebabCase(tech.id)}/`}>
-                <TechItem>
-                  <img srcSet={tech.logo.childImageSharp.fixed.srcSet} />
-                  <span className="name"> {tech.name} </span>
-                </TechItem>
-              </Link>
-            ))
-          }
+          {project.techstack.map(tech => (
+            <Link to={`/tech/${_.kebabCase(tech.id)}/`}>
+              <TechItem>
+                <img srcSet={tech.logo.childImageSharp.fixed.srcSet} />
+                <span className="name"> {tech.name} </span>
+              </TechItem>
+            </Link>
+          ))}
+        </div>
+      </div>
+      <div className="row">
+        <span className="label">Project Team</span>
```

Comment:
Do these have the similar styles as the tech stack? I don't see a separate component.